homeassistant/components/lyric/climate.py

```diff
@@ -211,6 +211,15 @@ def __init__(
         if LYRIC_HVAC_MODE_HEAT_COOL in device.allowed_modes:
             self._attr_hvac_modes.append(HVACMode.HEAT_COOL)
 
+        # TCC devices like the Lyric round do not have the Auto
+        # option in allowed_modes, but still support Auto mode
+        if (
```

Comment:
This should probably be an `elif` so HEAT_COOL is not added twice if TCC thermostats are added to allowed_modes in future

Reply:
Great point, I've made that change.

Reply:
Actually @timmo001, I ended up making that statement an or instead of elif to comply with ruff rules.